llvm-general/src/LLVM/General/Internal/FFI/Instruction.h

```diff
@@ -4,7 +4,7 @@
 #include "llvm/Config/llvm-config.h"
 
 #define LLVM_GENERAL_FOR_EACH_ATOMIC_ORDERING(macro) \
-	macro(NotAtomic) \
+    macro(NotAtomic)                 \
```

Comment:
Is this change intentional? In github it looks like it's just messing up the formatting.